docs/operations/example-dashboard-template.json

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
It seems this example dashboard doesn't include the WAL metrics you just added?

Reply:
Right. This example dashboard is for numaflow users to have a general view of the status of their pipelines. I didn't include WAL metrics in this specific dashboard because I consider WAL as implementation details of map-reduce, which don't necessarily need to be known by end users.

In the future , we can have a separate dashboard for numaflow developers which includes WAL metrics, hence I included the WAL dashboard template in the other PR for future reference.

Reply:
Yeah, definitely not a blocker for this PR. Thanks!

Reply:
WAL metrics can be merged with the reduce metrics once we have those available.